src/components/__tests__/SystemPreferences.spec.js

```diff
@@ -64,22 +64,22 @@ describe('SystemPreferences.vue', () => {
 
```

Comment:
A couple of lines before this there's an existing reference to `noChangesToApply` in `delete minimalProps.noChangesToApply` but this field doesn't appear anywhere else.

Reply:
This must have been leftover from a much older implementation.. removed.